ci/ray_ci/automation/update_version.py

```diff
@@ -1,28 +1,25 @@
-import os
-
 import click
 
 from ci.ray_ci.automation.update_version_lib import (
     get_current_version,
     update_file_version,
 )
 
-bazel_workspace_dir = os.environ.get("BUILD_WORKSPACE_DIRECTORY", "")
-
```

Comment:
can we also make this still working? like can we use this as the default value, and errors when it is empty?

Reply:
Oh yes that should be the default behavior for running locally

Reply:
updated to use bazel workspace dir as default when `root_dir`. If bazel workspace dir is empty, raise exception